src/country/country-info/country-info.component.ts

```diff
@@ -26,26 +38,31 @@ export class CountryInfoComponent implements OnInit, OnDestroy {
   public math: MathService;
   public countryInfoService: CountryInfoService;
   public countryInfoServiceSubscribe: Subscription;
-  public streetSettingsService: StreetSettingsService;
   public streetData: DrawDividersInterface;
   public streetServiceSubscribe: Subscription;
   public languageService: LanguageService;
   public device: BrowserDetectionService;
+  public store: Store<AppStateInterface>;
 
   public constructor(countryInfoService: CountryInfoService,
                      math: MathService,
-                     streetSettingsService: StreetSettingsService,
                      languageService: LanguageService,
-                     browserDetectionService: BrowserDetectionService) {
+                     browserDetectionService: BrowserDetectionService,
+                     store: Store<AppStateInterface>,
+                     private appEffects: AppEffects) {
     this.device = browserDetectionService;
     this.countryInfoService = countryInfoService;
     this.math = math;
-    this.streetSettingsService = streetSettingsService;
     this.isShowInfo = false;
     this.languageService = languageService;
+    this.store = store;
   }
 
   public ngOnInit(): void {
+    this.appEffects.getDataOrDispatch(this.store, AppEffects.GET_STREET_SETTINGS).then((data: any) => {
```

Comment:
wrong, you should use 
`this.streetData = store.select(function)` and `async` in html